action.yml

```diff
@@ -56,30 +52,28 @@ runs:
   args:
     - ${{ inputs.base_uri }}
     - ${{ inputs.cx_client_id }}
-    - ${{ inputs.cx_secret }}
-    - ${{ inputs.cx_token }}
+    - ${{ inputs.cx_client_secret }}
+    - ${{ inputs.cx_apikey }}
     - ${{ inputs.github_token }}
     - ${{ inputs.scan_types }}
     - ${{ inputs.project_name }}
     - ${{ inputs.archive_source }}
     - ${{ inputs.source_directory }}
     - ${{ inputs.additional_params }}
     - ${{ inputs.filter }}
-    - ${{ inputs.preset_name }}
 
   env:
     CX_BASE_URI: "${{ inputs.base_uri }}"
     CX_AST_ACCESS_KEY_ID: ${{ inputs.cx_client_id }}
-    CX_AST_ACCESS_KEY_SECRET: ${{ inputs.cx_secret }}
-    CX_TOKEN: ${{ inputs.cx_token }}
+    CX_AST_ACCESS_KEY_SECRET: ${{ inputs.cx_client_secret }}
```

Comment:
Do we need to have this env variables ? I think we we don't use it, we use the values directly from the secrects. Could you please check ?